src/OpenTelemetry.AutoInstrumentation/Logging/OtelLogging.cs

```diff
@@ -29,6 +29,7 @@ internal static class OtelLogging
 {
     private const string OtelDotnetAutoLogDirectory = "OTEL_DOTNET_AUTO_LOG_DIRECTORY";
     private const string NixDefaultDirectory = "/var/log/opentelemetry/dotnet";
+    private const int FileSizeLimitBytes = 10 * 1024 * 1024;
```

Comment:
Do we want to make it configurable? (separate PR for sure)

Reply:
There were additional things we wanted to make configurable (e.g enable logging), I intended to do that in a separate PR